3.3/Dockerfile.fedora

```diff
@@ -0,0 +1,52 @@
+FROM quay.io/fedora/s2i-base:39
```

Comment:
Fedora 39 does not have Ruby 3.3. Fedora 40 should be there ideally.

Reply:
You're right, that was for my testing (I've used COPR), as s2i-base:40 does not exist :).

I'll fix it, but it won't build though....

```
$ make build TARGET=fedora VERSIONS=3.3
make[1]: Entering directory '/home/lpcs/lpcsf-new/test/container/s2i-ruby-container'
mkdir -p 3.3/root
go-md2man -in "3.3/README.md" -out "3.3/root/help.1"
chmod a+r "3.3/root/help.1"
VERSION="3.3" SKIP_SQUASH=1 OS=fedora CLEAN_AFTER= DOCKER_BUILD_CONTEXT=. OPENSHIFT_NAMESPACES="2.0" CUSTOM_REPO="" REGISTRY="quay.io/" /usr/bin/env bash common/build.sh
Built versions are: 3.3
-> Version 3.3: building image from 'Dockerfile.fedora' ...
-> Pulling image quay.io/fedora/s2i-base:40 before building image from Dockerfile.fedora.
Trying to pull quay.io/fedora/s2i-base:40...
Error: initializing source docker://quay.io/fedora/s2i-base:40: reading manifest 40 in quay.io/fedora/s2i-base: manifest unknown
Pulling image quay.io/fedora/s2i-base:40 failed.

```

Reply:
we can have rawhide or latest and switch later to a released Fedora once it happens mayhaps.

Reply:
We will start to build s2i-base and s2i-core for f40 in short time.
https://github.com/sclorg/s2i-base-container/pull/288
Peto wanted to discuss it on tomorrow's container meeting first.
I think you can shift to f40 and when the s2i-base and core are ready we can rerun the tests.

Reply:
Fixed.